spec/slack-ruby-bot/commands/unknown_spec.rb

```diff
@@ -4,9 +4,16 @@
   def app
     SlackRubyBot::App.new
   end
+
+  def unknown_response
```

Comment:
The reason I pulled this out was because RuboCop was failing for the line being to long. I have restored the failure behaviour so you can see. This was also noted in the commit message but now they are squashed (strange to me). Let me know how you would like to resolve the line being too long - tweak RuboCop settings? Extract in another way?